packages/firebase_messaging/test/firebase_messaging_test.dart

```diff
@@ -137,6 +137,26 @@ void main() {
     firebaseMessaging.getToken();
     verify(mockChannel.invokeMethod('getToken'));
   });
+
+  test('deleteInstanceID', () {
+    firebaseMessaging.deleteInstanceID();
+    verify(mockChannel.invokeMethod('deleteInstanceID'));
+  });
+
+  test('autoInitEnabled', () {
+    firebaseMessaging.autoInitEnabled();
+    verify(mockChannel.invokeMethod('autoInitEnabled'));
+  });
+
+  test('setAutoInitEnabled', () {
+    bool enabled = true;
+    firebaseMessaging.setAutoInitEnabled(enabled);
+    verify(mockChannel.invokeMethod('setAutoInitEnabled', enabled));
```

Comment:
Could you verify explicitly here that `true` and `false` are called here?

```
verify(mockChannel.invokeMethod('setAutoInitEnabled', true));
```

Reply:
Is that what you mean?

```dart
    // assert that we havent called the method yet
    verifyNever(firebaseMessaging.setAutoInitEnabled(true));

    firebaseMessaging.setAutoInitEnabled(true);

    // assert we called the method with enabled = true
    verify(mockChannel.invokeMethod('setAutoInitEnabled', true));

    // assert that enabled = false was not yet called
    verifyNever(firebaseMessaging.setAutoInitEnabled(false));

    firebaseMessaging.setAutoInitEnabled(false);

    // assert call with enabled = false was properly done
    verify(mockChannel.invokeMethod('setAutoInitEnabled', false));
```

What I really would like to do is to call `autoInitEnabled()` to check its value after calling `setAutoInitEnabled`, but as both method implementations lie on the plugin, we don't have access on testing time - I think.

Reply:
Yes that look good to me.

Reply:
OK, it's done! 👍